cypress/src/plugins/removeVideosForPassedTests.tsx

```diff
@@ -0,0 +1,16 @@
+import del from 'del';
+
+const removeVideosForPassedTests = (on: Cypress.PluginEvents, _config: Cypress.PluginConfigOptions): void => {
```

Comment:
Do we need the `_config` parameter if it's not used?

Reply:
Fixed!